index.html

```diff
@@ -475,9 +475,12 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 <dfn>workerStart</dfn> attribute MUST return as follows:</p>
 <ol>
 <li>
-<p>If the current browsing or worker context's have an
+<p>If the current browsing or worker context have an
```

Comment:
Is there a better way to refer to this 'browsing or worker context'?

Reply:
We need to rewrite this, probably as the 
[active service worker](https://html.spec.whatwg.org/#concept-environment-active-service-worker) of the [context object](https://dom.spec.whatwg.org/#context-object)'s
[relevant settings object](https://html.spec.whatwg.org/#relevant-settings-object).

I'll do that in a followup.

Reply:
Cleaned it up here. PTAL